app/cdap/components/GridTable/components/Breadcrumb/styles.ts

```diff
@@ -25,4 +26,18 @@ export const useStyles = makeStyles({
     marginRight: '30px',
     marginLeft: '34px',
   },
+  breadcrumblabel: {
```

Comment:
this should be camel case
plz maintain across

Reply:
Done